src/v/kafka/client/test/fixture.h

```diff
@@ -19,7 +19,7 @@ namespace kc = kafka::client;
 
 class kafka_client_fixture : public redpanda_thread_fixture {
 public:
-    void restart() {
+    void restart(bool test_mode = false) {
```

Comment:
Given this is a test fixture, should this default to true?

Reply:
I thought the same thing; however, the original test has this flag unset. So to support backwards compatibility, I made it false.